client/packages/coldchain/src/Equipment/utils.ts

```diff
@@ -137,50 +137,67 @@ export const translateReason = (
 
 export const importEquipmentToCsvWithErrors = (
   assets: Partial<ImportRow & LineNumber>[],
-  t: TypedTFunction<LocaleKey>
+  t: TypedTFunction<LocaleKey>,
+  isCentralServer: boolean
 ) => {
   const fields: string[] = [
     t('label.asset-number'),
     t('label.catalogue-item-code'),
-    t('label.asset-notes'),
-    t('label.serial'),
-    t('label.installation-date'),
-    t('label.line-number'),
-    t('label.error-message'),
   ];
 
-  const data = assets.map(node => [
-    node.assetNumber,
-    node.catalogueItemCode,
-    node.notes,
-    node.serialNumber,
-    node.installationDate,
-    node.lineNumber,
-    node.errorMessage,
-  ]);
+  if (isCentralServer) {
+    fields.push(t('label.store'));
+  }
+
+  fields.push(t('label.asset-notes'));
```

Comment:
I wonder if all fields should be pushed rather than having asset number and catalogue-item-code in the original array?
I'm sure doesn't make a difference functionally, but makes it easier to read for me

Reply:
cool - have done. 🙏